### PR TITLE
Updates after review

### PR DESCRIPTION
**Pull request by EvsenevDev**
- [Fix] Debug State - Subscription
- Modify how our backup items are displayed on the backups page
- Google Drive Scope only using the App Data folder in Release
- Modify our backup restoration behavior such that our local database does not track our references to the old sync item
- Sync our `receipts.db`, even when the "Wifi only" setting is on

### CODE REVIEW

File: SmartReceipts/Services/Auto Backup/SyncService.swift [hunk 1]
```diff
@@ -134,11 +141,13 @@ class SyncService {
         syncProvider = SyncProvider.current
         syncService?.getCriticalSyncErrorStream().bind(to: syncErrorsSubject).disposed(by: bag)
     }
-    
+
     // MARK: - DB Handlers
     
     @objc private func didInsert(_ notification: Notification) {
         syncService?.syncDatabase()
+        if !canUploadReceipts { return }
```
Comment (wbaumann) [inline SmartReceipts/Services/Auto Backup/SyncService.swift]: Want to add a `Logger` message to this, stating something like `"Skipping insert due to lack of network availability"`

File: SmartReceipts/Services/Auto Backup/SyncService.swift [hunk 2]
```diff
@@ -150,6 +159,8 @@ class SyncService {
     
     @objc private func didUpdate(_ notification: Notification)  {
         syncService?.syncDatabase()
+        if !canUploadReceipts { return }
```
Comment (wbaumann) [inline SmartReceipts/Services/Auto Backup/SyncService.swift]: Same here

File: SmartReceipts/Services/Auto Backup/SyncService.swift (hunk 1 above)
Comment (EvsenevDev) [reply SmartReceipts/Services/Auto Backup/SyncService.swift]: @wbaumann it's not insert skip. Skipping sync images. `didInsert` calls after insert and sync receipts if needed.

Comment (wbaumann) [reply SmartReceipts/Services/Auto Backup/SyncService.swift]: Gotcha makes sense

**Pull request merged**